lib/robots/dor_repo/accession/publish.rb

```diff
@@ -14,7 +14,12 @@ def perform(druid)
           object_client = Dor::Services::Client.object(druid)
           obj = object_client.find
 
-          object_client.publish unless obj.is_a?(Cocina::Models::AdminPolicy)
+          return if obj.is_a?(Cocina::Models::AdminPolicy)
+
+          background_result_url = Dor::Services::Client.object(druid).publish
```

Comment:
I *think* you can reuse `object_client` here? So, `object_client.publish` instead.